services_benchmark_test.go

```diff
@@ -0,0 +1,261 @@
+package fiber
+
+import (
+	"context"
+	"testing"
+	"time"
+)
+
+func BenchmarkStartServices(b *testing.B) {
```

Comment:
All the benchmarks and tests should have an underscore "Benchmark_" and "Test_" for consistency with the other files.